### DISCUSSION TITLE
Customizing SVG graphics

### DISCUSSION THREAD

**Discussion by jimmyff** [zathras/jovial_svg#120] [Show and tell]
Here's a quick look at how I'm using Jovial SVG to customize SVGs!

https://youtu.be/B9rZwdGQ47E

(Skip to ~3:30 to go straight to the SVG stuff)

**Comment by zathras** [comment]
Cool!  Interesting usage - I gujess for a Ubuntu distribution, more or less targeted to an educational market?

I've been glad to see Ubuntu move toward Flutter for desktop apps.  Having an efficient language that encourages good software engineering practices (or at least enables them!) seems like a good idea.

BTW, `jovial_svg` got started because I wanted Aisleriot when I moved from Ubuntu to OS/X for my desktop machine a few years back.  I snagged the card images from the Linux aisleriot, but the SVG card assets didn't work with the Flutter SVG libraries I was able to find.  So I got annoyed, and wrote my own :-)
